### PR TITLE
changes to legal document section to reflect e-signature

### PR DESCRIPTION
**Pull request by gidsg**
https://trello.com/c/NutOcwCP/342-2-placeholder-triage-and-fix-e-signature-issues
Update text and logic for legal documents section to reflect e-signature. Text agreed with @NoraGDS 
Screenshot 2020-09-29 at 15 41 14

### CODE REVIEW

File: app/templates/frameworks/_agreement_returned_legal.html [hunk 1]
```diff
@@ -86,13 +97,21 @@ <h2 class="summary-item-heading">Legal documents</h2>
...
           {% else %}
             Download your countersigned framework agreement
           {% endif %}
         </a>
         <br/>
-        The agreement signed by the Crown Commercial Service
+        {% if e_signature_supported_framework %}
+        This agreement is signed by the Crown Commercial Service.
+        {% else %}
+        The agreement signed by the Crown Commercial Service.
```
Comment (NoraGDS) [inline app/templates/frameworks/_agreement_returned_legal.html:113]:
```suggestion
        The agreement signed by the Crown Commercial Service.
```
Delete this please.

Comment (gidsg) [reply app/templates/frameworks/_agreement_returned_legal.html:113]: @NoraGDS this only appears for non-e-signature frameworks and I guess gives them the difference between the two documents in that case.

**Comment by lfdebrux** [comment]
### My review checklist

I've:

- [ ] looked at the code
- [ ] tested locally
- [ ] run functional tests locally

**Review by lfdebrux — APPROVED** [review]
Works for me!

**Pull request merged**